docs/UserGuide.md

```diff
@@ -80,27 +77,27 @@ applications.
 
 ### Adding an application : `add`
 
-Adds an application to a company to the list.
+Adds an application to the job application list.
 
-**Format:** `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS`
+**Format:** `add c/COMPANY r/ROLE [d/DEADLINE] [s/STATUS] [i/INDUSTRY]`
 
 * Users must input a company `NAME`
-* Details of the company such as `PHONE_NUMBER`, `EMAIL` and `ADDRESS` are optional
+* Details of the company such as `DEADLINE`, `STATUS` and `INDUSTRY` are optional
 
 **Successful command:**
-print “Application to (Company name) has been added.”
+print “New job added: COMPANY; Role: ROLE; Status: STATUS; Deadline: DEADLINE; Industry: INDUSTRY"
 
 **Failed command:**
-print “Error: ” and error message for:
-
-* `NAME` is not in the command: “Name of company was not included in the command.”
+print "Invalid command format!
+add: Adds a job to the application book. 
+Parameters: c/COMPANY r/ROLE [Optional] d/DEADLINE s/STATUS i/INDUSTRY"
 
 **Examples:**
 
-* `add n/Microsoft p/98765432 e/microsoft@gmail.com a/182 Cecil St, #13-01, Singapore 069547`
-  Adds a company called microsoft, adds their phone number, email and address
-* `add n/The Coca-Cola Company e/cocacola@yahoo.com`
-  Adds a company called The Coca-Cola Company, adds their email
+* `add c/Google r/Software Engineer d/Dec 31 2030 1200 s/Approved, i/Technology`
+  Adds an application to a company Google, the role, deadline, status and industry of the job application
+* `add n/The Coca-Cola Company r/Quality Assurance Engineer`
+  Adds an application to a company The Coca-Cola Company and the role applied.
```

Comment:
I think I made similar edits, just ignore mine and use yours when resolving merge conflict!